reporter/tap.js

```diff
@@ -0,0 +1,77 @@
+(function() {
+
+function reporter( rep ) {
+	if ( rep === "tap" ) {
+		tapReporter();
+	}
+};
+
+reporter.print = function( val ) {
+	console.log( val );
+};
+
+QUnit.reporter = reporter;
```

Comment:
> This definitely doesn't belong in reporter/tap.js

definitely, I agree.

I'll review this, crossing with the efforts on the js-reporter and provide some good rebased thing.